lib/data_update_scripts/20200526181850_update_public_reaction_counts_again.rb

```diff
@@ -0,0 +1,10 @@
+module DataUpdateScripts
+  class UpdatePublicReactionCountsAgain
+    def run
+      ActiveRecord::Base.connection.execute("UPDATE articles SET public_reactions_count = positive_reactions_count, previous_public_reactions_count = previous_positive_reactions_count WHERE id < #{Article.count / 2}")
+      ActiveRecord::Base.connection.execute("UPDATE articles SET public_reactions_count = positive_reactions_count, previous_public_reactions_count = previous_positive_reactions_count WHERE id > #{Article.count / 2}")
```

Comment:
I think one of these two should have the `<=` or `>=` but I think it's fine anyway. Nice trick to split the table in two :)

Reply:
Wow we are definitely on the same page

Reply:
Ahh right haha

Reply:
Oh in addition to this, we should be storing the counts in a variable bc technically the counts could change between running each statement. I know its a small chance but its still a chance

Reply:
👍 updated